contributors/guide/issue-triage.md

```diff
@@ -71,8 +71,8 @@ If you see support questions on kubernetes-dev@googlegroups.com or issues asking
 support try to redirect them to Stack Overflow. Example response:
 
 ```code
-Please re-post your question to [Stack Overflow]
-(http://stackoverflow.com/questions/tagged/kubernetes).
+Please re-post your question to [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
```

Comment:
This moves the SO link one line up, because the preview looks awkward if copy pasted directly.